x/protorev/client/cli/tx.go

```diff
@@ -0,0 +1,139 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	"github.com/spf13/cobra"
+
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/osmosis-labs/osmosis/v13/x/protorev/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// CmdSetProtoRevAdminAccountProposal implements the command to submit a SetProtoRevAdminAccountProposal
+func CmdSetProtoRevAdminAccountProposal() *cobra.Command {
```

Comment:
We might not have to parse custom build message with new cli tools 

For reference: 
https://github.com/osmosis-labs/osmosis/blob/05a9fe4262a5fcc302e98b0ddf87d9f5f538b05e/x/concentrated-liquidity/client/cli/tx.go#L25-L51

Reply:
This is a pretty neat update (great to know for the last PR I am going to make with all of the CLI cmds for `x/protorev`). I like it. I'll refactor to use the new cli tools.

Reply:
Per convo, I refactored the proposals to route directly to protorev instead of gov - mirroring the txfees module proposal handlers.